packages/sources/mycryptoapi/test/integration/adapter.test.ts

```diff
@@ -0,0 +1,92 @@
+import { Requester } from '@chainlink/ea-bootstrap'
+import { assertError, assertSuccess } from '@chainlink/ea-test-helpers'
+import { AdapterRequest } from '@chainlink/types'
+import { makeExecute } from '../../src/adapter'
+
+describe('execute', () => {
```

Comment:
We should add `nock` based integration tests in here so we're testing more than just the validation logic, see #820 for reference. If you need help with this feel free to reach out to me :D